script/Ciel/Day 13/13_01_01_CIEL13_4.txt

```diff
@@ -26,7 +26,7 @@ But even that is at its limit.
 [sha:b746d22c795f4439e41f5db7f8fde6ae637fe7de]{
 -- Page 311, Offset 34099.
 -- 息があがったのか、千切れかけた左手を振る事もできなくなったのか。
-Perhaps it's because I'm out of breath, or maybe it's because I can't even wave my left hand that's torn to pieces.
+Perhaps it's because I'm out of breath, or maybe it's because I can't move my left hand anymore since it's been torn to pieces.
```

Comment:
Maybe it's because I'm out of breath, or maybe it's because I've lost any feeling in my shredded left hand.